designs/logging_facade/logging_facade.md

```diff
@@ -0,0 +1,80 @@
+# Using SLF4J Logging Facade
+Currently, as of [osc-core / 02b8024](https://github.com/opensecuritycontroller/osc-core/commit/02b80247faa29b069d2b6082f2e866a2f71b0f20),
+OSC uses the log4j logging framework directly. The goal is to add flexibility by having all the code reference 
+the **slf4j** facade only. Another logging framework may be plugged in by the user.
+
+## Background
+
+The SLF4j facade provides a common API, inluding logging levels: **error**, **warn**, **info**, **debug** and (since 1.4.0) **trace**. It detects the presence of one of several logging frameworks and uses it. 
+
+We want to use be able to use it throughout the project because:
+- We shall then be able to easily replace the logging framework behind the facade.
+- The implementation log4j, version 1.2, currently used, is obsolete.
+- A specific requirement is that plugin implementations be able to use our slf4j log configuration without any dependence on any of our packages. 

```

Comment:
dependence -> dependency, I recommend looking for typos in general with some auto-correct